staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go

```diff
@@ -164,10 +148,16 @@ func (c *APIServiceRegistrationController) addAPIService(obj interface{}) {
 	c.enqueue(castObj)
 }
 
-func (c *APIServiceRegistrationController) updateAPIService(obj, _ interface{}) {
-	castObj := obj.(*apiregistration.APIService)
-	glog.V(4).Infof("Updating %s", castObj.Name)
-	c.enqueue(castObj)
+func (c *APIServiceRegistrationController) updateAPIService(old, new interface{}) {
+	oldObj := old.(*apiregistration.APIService)
```

Comment:
revert this change, resync is a backstop to catch any inconsistencies that creep in, and doesn't happen frequently

Reply:
make sense. But I doubt why some other controllers compare pod ResourceVersion .

Reply:
After some deep re-thought and look into the code, I think if APIService periodic resync with same resource version, the sync loop will make a dry-run, since it get `APIService` from local cache. Which still can not `catch any inconsistencies that creep in`.